pages/_app.tsx

```diff
@@ -27,83 +27,99 @@ import debug from 'debug';
 import Script from 'next/script';
 import Banner from 'modules/app/components/layout/header/Banner';
 import bannerContent from 'modules/home/data/bannerContent.json';
+import useDelegatedToExpired from 'modules/delegation-renewal/hooks/useDelegatedToExpired';
 const vitalslog = debug('govpo:vitals');
 
 const Web3ReactProviderDefault = dynamic(() => import('../modules/web3/components/DefaultProvider'), {
   ssr: false
 });
 export const reportWebVitals = vitalslog;
 
-const MyApp = ({ Component, pageProps }: AppProps): React.ReactElement => {
+const App = ({ Component, pageProps }: AppProps): React.ReactElement => {
   ethers.utils.Logger.setLogLevel(ethers.utils.Logger.levels.ERROR);
 
   const activeBannerContent = bannerContent.find(({ active }) => active === true);
+  const hasDelegateExpiring = useDelegatedToExpired();
 
+  return (
+    <ThemeProvider theme={theme as any}>
+      <NextNprogress
+        color="#1aab9b"
+        startPosition={0.3}
+        stopDelayMs={200}
+        height={3}
+        showOnShallow={true}
+        options={{ showSpinner: false }}
+      />
+      <AccountProvider>
+        <BallotProvider>
+          <HeadComponent />
+          {process.env.NODE_ENV === 'production' && (
+            <Script
+              data-goatcounter="https://dux-makerdao.goatcounter.com/count"
+              async
+              src="//gc.zgo.at/count.js"
+              strategy="afterInteractive"
+            />
+          )}
+          <CookiesProvider disabled={false}>
+            <AnalyticsProvider>
+              <SWRConfig
+                value={{
+                  // default to 60 second refresh intervals
+                  refreshInterval: 60000,
+                  revalidateOnMount: true,
+                  fetcher: url => fetchJson(url)
+                }}
+              >
+                <Global
+                  styles={{
+                    '*': {
+                      WebkitFontSmoothing: 'antialiased',
+                      MozOsxFontSmoothing: 'grayscale'
+                    }
+                  }}
+                />
+                {activeBannerContent && <Banner content={activeBannerContent.content} />}
+                {hasDelegateExpiring && (
+                  <Banner
+                    variant="warning"
+                    content={
+                      <Text sx={{ fontWeight: '400' }}>
+                        You delegated MKR to a contract that is going to expire or has expired. Please migrate
+                        your MKR to a renewed Delegate Contract
+                      </Text>
+                    }
+                  />
+                )}
```

Comment:
Maybe a separate component for this even though it's small, so we don't clutter up `_app.tsx` with copy and styles and what not.